plugins/gateway/tyk/deploy/patch-deployment.yaml

```diff
@@ -15,7 +15,7 @@ spec:
             fieldRef:
               fieldPath: metadata.namespace
         - name: APICLARITY_HOST
-          value: "apiclarity.apiclarity:9000"
+          value: "apiclarity-apiclarity.apiclarity:9000"
```

Comment:
should be configured also with an env var

Reply:
Do you mean in values file? This PR is to fix the current behavior of the plugin deployments. Once we support plugin deployments with helm we can  add env var

Reply:
env var from script, like `UPSTREAM_TELEMETRY_HOST_NAME`